### ISSUE TITLE
Low contrast hyperlink in the dark background sections

### ISSUE THREAD

**Issue by raunaqgupta** [open-telemetry/opentelemetry.io#1866]
**Desired feature or idea.**
Improve hyperlink color when inside dark background sections. They currently also fail all WCAG levels.

![image](https://user-images.githubusercontent.com/623670/195948780-22277d77-51f0-42e6-a131-87acd24046d2.png)

**Comment by cartermp** [comment]
@chalin and @Arhell any thoughts?

**Comment by melbgirl** [comment]
+1 for making a fix to this issue: improving the accessibility of the site helps make OTel more inclusive to the community.

**Comment by melbgirl** [comment]
@cartermp if we decide we do want to make a change, I'm happy to take this issue.

**Comment by melbgirl** [comment]
@raunaqgupta do you recommend the site handling the two modes (ie behaving one way for dark mode, behaving differently for light mode), or simply changing the URL coloring so that it can be perceived in either dark or light mode?

**Comment by cartermp** [comment]
I would recommend just updating colors so contrast is the so bad. IIRC we don't have a dark mode for the site.

**Comment by raunaqgupta** [comment]
https://github.com/open-telemetry/opentelemetry.io/blob/f5fc3c0a8d965794e4500bff85656511cf107c17/assets/scss/_styles_project.scss#L122

Here's the file/style where the `a` color is specifically being set for dark and secondary sections. My suggestions are to:

1. Separate dark and secondary
2. Update hyperlink for dark
3. I personally feel the secondary section is too yellow and can be toned down a bit
4. Update the hyperlink color (if needed after 3) for the secondary section.

I'll send a PR soon.

**Comment by chalin** [comment]
PR merged, thanks! I'll submit a followup PR for the remaining issues: #1916, doesn't fix the contrast on the Registry page, for example.

**Comment by chalin** [comment]
Here's the PR which, if y'all agree, should close this issue:

- #1937
